src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -853,6 +856,10 @@ public void execute() throws GitException, InterruptedException {
                     throw new GitException("Error launching git whatchanged",e);
                 }
             }
+
+			public void listMerges(boolean flag) {
```

Comment:
Please use consistent formatting (spaces vs. tabs) so that the alignment of text blocks in the file doesn't become any worse than it already is.  I realize that the text formatting is already inconsistent, but I would prefer to not have new inconsistencies added.